murenn/dtcwt/graph.py

```diff
@@ -0,0 +1,51 @@
+import torch
+import murenn
+
+
+class MURENN_GRAPH(torch.nn.Module):
+    def __init__(
+        self,
+        J=8,
```

Comment:
no default values for keyword arguments `J`, `T`, `Q`

use required keyword arguments
`def __init__(self, *, J, Q, T, in_channels, padding_mode='symmetric')`